doc/website/sidebars.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
       ],
       collapsed: false,
     },
+    'ci',
```

Comment:
After getting-started? Move it to `Misc` now and we'll reference it from other pages.